### PR TITLE
fix chameleon icons not appearing properly upon examine

### PR DESCRIPTION
**Pull request by ghost**
## About the Pull Request

Refs: https://github.com/Foundation-19/Foundation-19/issues/625

Chameleon items don't show the correct icon when examining a mob -- this fixes that.

Didn't need to port /tg/'s `getflaticon`, ours is fine. 

## Why It's Good For The Game

Traitors not getting outed from a bug is good.

## Changelog

:cl:
fix: chameleon item icons not appearing properly when examining a mob
/:cl:

### CODE REVIEW

**Comment by Tsurupeta** [comment]
Getflaticon is very heavy proc, putting it into every examine is a really great way to lag the server down.

**Comment by checkraisefold** [comment]
merge this so i can spam examine to crash the server pls

File: code/game/objects/items.dm [hunk 1]
```diff
@@ -859,9 +859,9 @@ modules/mob/living/carbon/human/life.dm if you die, you will be zoomed out.
 
 /obj/item/proc/get_examine_line(mob/user)
 	if(blood_color)
-		. = SPAN_WARNING("[icon2html(src, user)] [gender==PLURAL?"some":"a"] <font color='[blood_color]'>stained</font> [src]")
+		. = SPAN_WARNING("[icon2html(getFlatIcon(src), user)] [gender==PLURAL?"some":"a"] <font color='[blood_color]'>stained</font> [src]")
 	else
-		. = "[icon2html(src, viewers(get_turf(src)))] \a [src]"
+		. = "[icon2html(getFlatIcon(src), user)] \a [src]"
```
Comment (Vizoplez) [inline code/game/objects/items.dm:864]: this WILL crash the server

Comment (MACIEKBAKI) [reply code/game/objects/items.dm:864]: and if it does not then it will cause quite a bit of lag... if (so, when) people spam examine

Comment (ghost) [reply code/game/objects/items.dm:864]: Yeah, I'll figure out a work-around

**Pull request closed without merging**